packages/presentation/lib/utils/dimensions.dart

```diff
@@ -0,0 +1,50 @@
+class AppSizes {
+  AppSizes._();
+  static const double size0 = 0;
+  static const double size0point5 = 0.5;
```

Comment:
try to avoid fractional sizes. use 0 or 1. half of the pixel/unit is not notable at UI